operator/test/manifests/test/tekton-chains/public-key.yaml

```diff
@@ -20,4 +20,4 @@ spec:
                 set -x
                 PUBLIC_KEY=$(oc get secret public-key -n tekton-chains -o jsonpath='{.data.cosign\.pub}')
                 echo "$PUBLIC_KEY" | base64 -d
-  serviceAccountName: pipeline
+  serviceAccountName: appstudio-pipeline
```

Comment:
Do we want to set this SA as global -> https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/openshift-pipelines/tekton-config.yaml#L12 ?

Reply:
I would say yes - see my comment https://github.com/openshift-pipelines/pipeline-service/pull/629#issuecomment-1531520847